backstop_data/engine_scripts/puppet/price-set/add-price-field-modal.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+module.exports = async (engine, scenario, viewport) => {
+  await require('./view-and-edit-price-fields')(engine, scenario, viewport) ;
```

Comment:
There is a space before `;` that should not exist. Do we have a linter enabled?

Reply:
fixed!